src/config/mod.rs

```diff
@@ -6,19 +6,19 @@ use std::{
     path::PathBuf,
 };
 
-use async_trait::async_trait;
 use indexmap::IndexMap; // IndexMap preserves insertion order, allowing us to output errors in the same order they are present in the file
```

Comment:
This comment may be more topical co-located with the usage of IndexMap on ln114

Reply:
Just gonna remove it entirely, I think. We use `IndexMap` all over the place, it seems weird to only document the "why" of why we use it in this one particular place but not the others.